src/string_enums/enum_base.hpp

```diff
@@ -55,6 +55,20 @@ struct enum_base : public T
 		return std::nullopt;
 	}
 
+	/**
```

Comment:
It was added in a previous PR, but I'm wondering if there should be a bounds-check in get_string() to handle `get_string(ENUM_MAX)`, which I think would be undefined behavior accessing beyond the end of `T::values`.

Reply:
Should it just return a default like:
```
if(key == T::type::ENUM_MAX) {
	return "ENUM_MAX";
}
```
or something else?

Reply:
`ENUM_MAX` is an invalid value, so I'd go with either an empty string or nullopt.

Reply:
It doesn't return an optional currently, so I guess an empty string would be simpler.

Reply:
Done.

Reply:
Another option also being to use `at(int)`, which would throw instead of resulting in UB.